### PR TITLE
(#909) Trying to get Travis build working

### LINKED ISSUE #909: MacOS build is broken
introduced in #908

### PR DESCRIPTION
**Pull request by TheKnarf**
Fixes #909

### CODE REVIEW

**Comment by TheKnarf** [comment]
Okay this didn't work, hmm...

**Comment by rexim** [comment]
@TheKnarf what's funny it was working like that for quite some time and one day suddenly stopped. So it broke by itself.

**Comment by TheKnarf** [comment]
🎉

File: .travis.yml [hunk 1]
```diff
@@ -4,14 +4,16 @@ addons:
...
       - sdl2
       - cmake
+    update: true
 os:
   - osx
 compiler:
   - clang
   - gcc
 script:
+  - brew info sdl2 # for debug test
```
Comment (rexim) [inline .travis.yml]: Do we still need that?

Comment (TheKnarf) [reply .travis.yml]: No

File: .travis.yml [hunk 2]
```diff
@@ -4,14 +4,16 @@ addons:
...
   - mkdir build/
   - cd build/
-  - cmake ..
+  - cmake -DSDL2_INCLUDE_DIR=/usr/local/include ..
```
Comment (rexim) [inline .travis.yml]: Do we still need that?

Comment (TheKnarf) [reply .travis.yml]: Maybe? Probably not?

**Comment by rexim** [comment]
@TheKnarf thank you so much for the contribution!

**Pull request merged**